### ISSUE TITLE
Line break in org-mode when export to html don't work

### ISSUE THREAD

**Issue by tarlaw** [hlissner/doom-emacs#3654]
Use emacs with org-mode 9.3.7 export this

```org
This is a test for \\
export new line.
```
as html
```html
<p>
This is a test for <br />
export new line.
</p>
```

But in doom emacs the line break don't work

```html
<p>
This is a test for
export new line.
</p>
```

And `#+OPTIONS: \n:t` also don't work in doom.

**Comment by hlissner** [comment]
In the latest version of org-mode I believe they renamed the option `line-break`. Try

```org
#+OPTIONS: line-break:t
```

**Comment by tarlaw** [comment]
The problem solved after reinstall doom, it should be something wrong in my config. 
Sorry for issue without full test.

**Comment by tarlaw** [comment]
The issue reappear after I install `input/chinese` module, I think it caused by `org-html-paragraph` hack.
To me, the default org export logic is enough even for Chinese, the hack seemed not make sense, but it just by design.
I just need fcitx, so I install it separately.

**Comment by hlissner** [comment]
Ah, good catch. I've fixed the advice in ffbd7b6. Thanks for letting me know!
